charts/resident/values.yaml

```diff
@@ -424,3 +424,34 @@ istio:
     - istio-system/public
     - istio-system/internal
   prefix: /resident/v1
+  corsPolicy:
+    allowOrigins:
+      - prefix: https://resident-ui.sandbox.xyz.net
```

Comment:
change https://resident-ui.sandbox.xyz.net 
to https://resident.sandbox.xyz.net